src/win/pipe.c

```diff
@@ -925,6 +921,7 @@ int uv_pipe_accept(uv_pipe_t* server, uv_stream_t* client) {
     req->next_pending = NULL;
     req->pipeHandle = INVALID_HANDLE_VALUE;
 
+    server->handle = INVALID_HANDLE_VALUE;
```

Comment:
Shouldn't server->handle be the actual server'ss pipe handle? Why are we null-ing it now?

Reply:
The server doesn't have a permanent handle of its own, so it just temporarily borrows the handle for a future client. We just gave away our handle (`req = server->pipe.serv.pending_accepts; pipe_client->handle = req->pipeHandle;`), so we want to also drop our reference to it at the same time for clarity (`uv_pipe_queue_accept` will allocate a new one, if we're not closing).